helm/examples/filestore/README.md

```diff
@@ -0,0 +1,47 @@
+# HPCC FileStore storage
+
+## Directory contents
+
+### FileStore Instance
+Filestore instances are fully managed NFS file servers on Google Cloud for use with applications running on Compute Engine virtual machines (VMs) instances or Google Kubernetes Engine clusters. You can find more information here: https://cloud.google.com/filestore/docs
+
+To create a Filestore instance: https://cloud.google.com/filestore/docs/creating-instances
+If possible create a Filestore instance in the same zone
+
+Here is a sample code
+```console
+gcloud filestore instances create -q filestore1 \
+    --project=<GCP Project ID> \
+    --zone=us-east1-b \
+    --tier=STANDARD \
+    --network=name=default \
+    --file-share=name="hpccdata",capacity=1TB
+```
+
+You will need instance ip and file-share name (hpccdata) for further configuration
```

Comment:
always ip only or is there a generated host name available after creating a filestore instance?

Reply:
For Filestore instance only ip is available. Not sure in future if it will support FQDN.